src/sm/sm-common.c

```diff
@@ -524,6 +524,7 @@ sm_encrypt_des_cbc3(struct sc_context *ctx, unsigned char *key,
 		free(data);
 		LOG_TEST_RET(ctx, SC_ERROR_OUT_OF_MEMORY, "SM encrypt_des_cbc3: failure");
 	}
+	memset(*out, 0, data_len + 8);
```

Comment:
would not it be easier to use `calloc()` instead of `malloc()` on line 522?

Reply:
Fixed, thanks for review.